(pp) real_pick_delCol.py

```diff
@@ -22,7 +22,15 @@
 # Autoencoder Features (from pickle Files)
 # Location of the Real-Apple data already split and downsampled
 # location = 'C:/Users/15416/PycharmProjects\PickApp\data\Real Apples Data/real_data_stage1/'
-location = 'C:/Users/15416/Box/Learning to pick fruit/Apple Pick Data/RAL22 Paper/4_proxy_winter22_x5/PICK/pp1_split_x45cols/'
+
+main = 'C:/Users/15416/Box/Learning to pick fruit/Apple Pick Data/RAL22 Paper/'
+dataset = '5_real_fall21_x1/'
+subfolder = 'pp1_split_x45cols/'
+
+stage = 'PICK/'
+
+location = main + dataset + stage + subfolder
```

Comment:
This is another case where you could, for instance, have a user run the file in "main" and specify dataset and subfolder.